source/deploy/quick-start-evaluation.rst

```diff
@@ -15,26 +15,61 @@ Deployment options
     * An Azure subscription
     * Basic familiarity with Azure Portal
 
-    **Steps to deploy Mattermost on Azure:**
+    The Solution Template sets up the following components:
 
-    1. Visit the `Mattermost, Collaboration for Mission-Critical Work (VM) <https://azuremarketplace.microsoft.com/de-de/marketplace/apps/mattermost.mattermost-all-in-one?tab=overview>`_
+    1. **Virtual Machine (Ubuntu):** Hosts the Mattermost application and its associated database, providing the computing environment required to run the system.
+    2. **Public IP Address:** Allows access to the Mattermost workspace over the public internet, enabling users to connect to the platform securely.
+    3. **Network Interface:** Facilitates communication between the Virtual Machine and other Azure resources through a defined network layer.
+    4. **Network Security Group:** Acts as a virtual firewall, controlling inbound and outbound traffic to protect the deployed resources from unauthorized access.
+    5. **Disk:** Provides persistent storage mounted to the Virtual Machine for storing files generated by Mattermost, including uploaded documents and system configurations.
+    6. **Virtual Network:** Creates a secure, isolated network environment for the deployed resources, enabling private and controlled connectivity between infrastructure components.
```

Comment:
You could argue that given this is for quick start and trials for the most part that this information is probably not necessary. I'm 0/5 on keeping it.